src/Common/Crypto/CryptoGen.php

```diff
@@ -405,16 +440,17 @@ public function aes256Decrypt_mycrypt($sValue)
      *  key set from database, collect key set from drive, decrypt key set from drive using the database
      *  key; caching the key will bypass all these steps).
      *
-     * @param  string  $version   This is the number/version of they key.
-     * @param  string  $sub       This is the sublabel of the key
-     * @param  string  $keySource This is the source of the standard keys. Options are 'drive' and 'database'
+     * @param string $version     This is the number/version of they key.
+     * @param string $sub         This is the sublabel of the key
+     * @param string $keySource   This is the source of the standard keys. Options are 'drive' and 'database'
      *                            The 'drive' keys are stored at sites/<site-dir>/documents/logs_and_misc/methods
      *                            The 'database' keys are stored in the 'keys' sql table
      * @return string             Returns the key in raw form.
      */
     private function collectCryptoKey($version = "one", $sub = "", $keySource = 'drive')
     {
         // Check if key is in the cache first (and return it if it is)
+        // TODO: I believe method used here is deprecated in PHP 8.2 and will throw error in PHP 9.0
```

Comment:
yep, you are correct in why i added the AllowDynamicProperties attribute :)
I'll look into a better long term solution.

Reply:
This will fix it:
https://github.com/openemr/openemr/pull/7342

Reply:
btw, I'll bring in my PR after you get your PR in (to avoid conflcts in your PR)